rig.js

```
@@ -498,87 +496,64 @@ class RigManager {
-      peerRig.inputs.leftGamepad.quaternion.fromArray(leftGamepadQuaternion);
-      peerRig.inputs.leftGamepad.pointer = leftGamepadPointer;
-      peerRig.inputs.leftGamepad.grip = leftGamepadGrip;
-
-      peerRig.inputs.rightGamepad.position.fromArray(rightGamepadPosition);
-      peerRig.inputs.rightGamepad.quaternion.fromArray(rightGamepadQuaternion);
-      peerRig.inputs.rightGamepad.pointer = rightGamepadPointer;
-      peerRig.inputs.rightGamepad.grip = rightGamepadGrip;
-
-      peerRig.setFloorHeight(floorHeight);
-      for (let i = 0; i < 2; i++) {
-        peerRig.setHandEnabled(i, handsEnabled[i]);
-      }
-      peerRig.setTopEnabled(topEnabled);
-      peerRig.setBottomEnabled(bottomEnabled);
-      peerRig.direction.fromArray(direction);
-      peerRig.velocity.fromArray(velocity);
-      peerRig.jumpState = jumpState;
-      peerRig.jumpTime = jumpTime;
-      peerRig.flyState = flyState;
-      peerRig.flyTime = flyTime;
-      peerRig.useTime = useTime;
-      peerRig.useAnimation = useAnimation;
-      peerRig.sitState = sitState;
-      peerRig.sitAnimation = sitAnimation;
-      peerRig.danceState = danceState;
-      peerRig.danceTime = danceTime;
-      peerRig.danceAnimation = danceAnimation;
-      peerRig.throwState = throwState;
-      peerRig.throwTime = throwTime;
-      peerRig.crouchState = crouchState;
-      peerRig.crouchTime = crouchTime;
-
-      peerRig.textMesh.position.copy(peerRig.inputs.hmd.position);
-      peerRig.textMesh.position.y += 0.5;
-      peerRig.textMesh.quaternion.copy(peerRig.inputs.hmd.quaternion);
-      localEuler.setFromQuaternion(peerRig.textMesh.quaternion, 'YXZ');
-      localEuler.x = 0;
-      localEuler.y += Math.PI;
-      localEuler.z = 0;
-      peerRig.textMesh.quaternion.setFromEuler(localEuler);
-
-      peerRig.rigCapsule.position.copy(peerRig.inputs.hmd.position);
-    }
+  setPeerAvatarPose(player) {
+    const peerRig = this.peerRigs.get(player.id);
+    if (!peerRig) return;
```

Would there ever be a player without a rig? If that happens it seems like it's a bug with not setting a rig when we create a player.

I believe the fake rigs can be generated with `new Avatar(null, {})`, synchronously, so this error condition should never happen. IMO if this `if` statement throws an internal error it would be better than silently skipping.

---

I removed this check after a bunch of testing without it.